presto-main/src/test/java/com/facebook/presto/type/TestTypeRegistry.java

```diff
@@ -14,7 +14,6 @@
 package com.facebook.presto.type;
 
 import com.facebook.presto.block.BlockEncodingManager;
-import com.facebook.presto.metadata.CastType;
```

Comment:
verify why this line is auto optimized...

Reply:
This seems unchanged in my commit?

Reply:
This line should not be here. I removed the commit that introduces CastType.